.github/workflows/release.yml

```diff
@@ -0,0 +1,60 @@
+name: Release CI
+
+on:
+  push:
+    branches:
+      - master
+
+env:
+  PLATFORM: linux/amd64
+  DOCKER_REPO: renovate/java
+  DOCKER_CACHE: java
+  # https://github.com/chalk/supports-color/issues/106
+  FORCE_COLOR: true
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    timeout-minutes: 10
+
+    steps:
+      - name: Set up Docker Buildx
+        id: buildx
+        uses: crazy-max/ghaction-docker-buildx@v1
+        with:
+          version: v0.3.1
+
+      - uses: actions/checkout@v2
+
+      - name: docker cache
+        id: docker-cache
+        uses: actions/cache@v1
+        with:
+          path: .cache
+          key: ${{ runner.os }}-docker-${{ hashFiles('Dockerfile') }}
+
+      - name: Docker registry login
+        run: echo "${{ secrets.DOCKER_PASSWORD }}" | docker login -u ${{ secrets.DOCKER_USERNAME }} --password-stdin
+
+      - name: Build the Docker images
+        run: |
+          cache_from=$([ -d .cache ] && echo -n '--cache-from=type=local,src=.cache' || echo -n '')
+          docker buildx build \
+            --load \
+            --tag ${DOCKER_REPO} --tag ${DOCKER_REPO}:11 \
```

Comment:
Do we want to put this `11` into an env so it's more explicit/obvious?

Reply:
It's just for now, when i've done the new build action, this will be retrived from branch name, as discussed on slack

Reply:
ok